tests/tests_fabric/conftest.py

```diff
@@ -54,6 +54,7 @@ def restore_env_variables():
         "RANK",  # set by DeepSpeed
         "POPLAR_ENGINE_OPTIONS",  # set by IPUStrategy
         "CUDA_MODULE_LOADING",  # leaked since PyTorch 1.13
+        "CRC32C_SW_MODE",  # leaked by tensorboardX
```

Comment:
Fabric tests with tensorboard. Either the comment is not correct or this shouldn't be necessary

Reply:
What does it matter. It is the same as in pytorch's version of conftest. 
Whether the tests run one way or another depends on which package is installed. Either you have TB installed, or you have TBX installed or both, or none. The tests should run/skip accordingly no matter which configuration.